src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.exception.DukeException;
+
+/**
+ * CS2103T Individual Project AY 21/22 Sem 1
+ * Project Duke
+ *
+ *
+ * Description:
+ * Abstract class that encapsulates a command inputted by the user
+ *
+ * @author Keith Tan
+ */
+public abstract class Command {
+
+    /**
+     * Returns a boolean of whether current command is the 'bye' command
+     *
+     * @return boolean returns boolean of whether current command is 'bye' command
+     */
```

Comment:
Perhaps the name of the method can be changed to isByeCommand to improve readability